python-frontend/src/main/java/org/sonar/python/semantic/v2/converter/PythonTypeToDescriptorConverter.java

```diff
@@ -132,6 +132,7 @@ private static Descriptor convert(String moduleFqn, String parentFqn, String sym
     var metaclassFQN = type.metaClasses()
       .stream()
       .map(metaClass -> typeFqn(moduleFqn, metaClass))
+      .filter(Objects::nonNull)
```

Comment:
This here is yet again an example of pitfalls when using streams this way.
You missed this edge case during implementation, and I missed it during review. The coverage was 100%, yet we experienced an NPE after merging.
I think we need to make progress and I don't want to refactor just for the sake of it (maybe `typeFqn` should return Optional?), but I must admit I'm a bit annoyed that we fall into similar traps so often.